metadata/src/test/java/org/apache/kafka/controller/QuorumControllerMetricsTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.controller;
+
+import com.yammer.metrics.core.MetricsRegistry;
+import org.apache.kafka.common.utils.Utils;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class QuorumControllerMetricsTest {
+    @Test
+    public void testKafkaControllerMetricNames() {
+        String expectedGroup = "kafka.controller";
+        String expectedType = "KafkaController";
+        Set<String> expectedMetricNames = Utils.mkSet(
+            "ActiveControllerCount",
+            "GlobalTopicCount",
+            "GlobalPartitionCount",
+            "OfflinePartitionsCount",
+            "PreferredReplicaImbalanceCount");
+        Set<String> missingMetrics = getMissingMetricNames(expectedMetricNames, expectedGroup, expectedType);
+        assertEquals(Collections.emptySet(), missingMetrics, "Expected metrics did not exist");
+    }
+
+    @Test
+    public void testControllerEventManagerMetricNames() {
+        String expectedGroup = "kafka.controller";
+        String expectedType = "ControllerEventManager";
+        Set<String> expectedMetricNames = new HashSet<>(Arrays.asList(
+            "EventQueueTimeMs",
+            "EventQueueProcessingTimeMs"));
+        Set<String> missingMetrics = getMissingMetricNames(expectedMetricNames, expectedGroup, expectedType);
+        assertEquals(Collections.emptySet(), missingMetrics, "Expected metrics did not exist");
```

Comment:
nit: When this assert failed, we'll see the error messages:
`Expected metrics did not exist` ==> expected: `emptySet`, but was: `missingMetrics`

I think we should change the error messages, ex:
`Expect no missing metrics` ==> expected: `emptySet`, but was: `missingMetrics`